build.sbt

```diff
@@ -43,13 +41,8 @@ libraryDependencies ++= Seq(
     exclude("org.typelevel", "cats-laws_2.11")
     exclude("org.typelevel", "cats-kernel-laws_2.11"),
   "ch.qos.logback" % "logback-classic" % "1.2.1" % Provided,
-  "org.webjars" % "swagger-ui" % "2.2.2" % Provided,
-  "io.spray" %% "spray-routing" % sprayV % Provided,
-  "io.spray" %% "spray-http" % sprayV % Provided,
-  "io.spray" %% "spray-can" % sprayV % Provided,
   "com.typesafe.akka" %% "akka-actor" % akkaV % Provided,
-  //---------- Test libraries -------------------//
```

Comment:
Cromwell is just ahead of the curve of everyone else. Cromwell used to have [the same](https://github.com/broadinstitute/cromwell/blob/0.18/build.sbt#L74). Lenthall (above),  [wdltool](https://github.com/broadinstitute/wdltool/blob/0.9/build.sbt#L38), [wdl4s](https://github.com/broadinstitute/wdl4s/blob/0.10/build.sbt#L51), and [centaur](https://github.com/broadinstitute/centaur/blob/c377c658ec47cf3c0acf7966d3f56eaa27362d2c/build.sbt#L27) all use this "old" style.